modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
Was `formatFileSize.js` previously conforming to spec? If that's the case then we could either do `filesize(bytes, {base: 10}).toUpperCase()` and export that here, or I could revert this file and have places that are using the filesize module use this instead (and remove the dependency on filesize)